### DISCUSSION TITLE
FileUpload image preview for filename without directory

### DISCUSSION THREAD

**Discussion by rezakho** [filamentphp/filament#10561] [Help]
How to preview image when directory not saved in table field?
exam: 
db table field value: qaOQ0GPv7YQ7caAXIjXpI5Ml3bZWpFW4hT9J8LxA.jpg
storage (url) path: 215\qaOQ0GPv7YQ7caAXIjXpI5Ml3bZWpFW4hT9J8LxA.jpg

I know how directory() works but I don't store the value of dir in the table.

**Comment by leandrocfe** [comment]
why you aren't saving it?

**Reply by rezakho** [reply]
The project and db are legacy and in use!
